main.py

```diff
@@ -565,3 +548,25 @@ def handle_stats_reply(self, msg, switch):
             event = KytosEvent('amlight/flow_stats.flows_updated')
             event.content['switch'] = switch.dpid
             self.controller.buffers.app.put(event)
+
+    @listen_to('kytos/of_core.flow_stats.received')
+    def on_stats_received(self, event):
+        """Capture flow stats messages for OpenFlow 1.3."""
+        self.handle_stats_received(event)
+
+    def handle_stats_received(self, event):
+        """Handle flow stats messages for OpenFlow 1.3."""
+        switch = event.content['switch']
+        if 'replies_flows' in event.content:
+            replies_flows = event.content['replies_flows']
+            self.handle_stats_reply_received(switch, replies_flows)
+
+    def handle_stats_reply_received(self, switch, replies_flows):
+        """Iterate on the replies and set the generic flows"""
+        switch.generic_new_flows.extend(replies_flows)
```

Comment:
This statement on line 566 will crash, `Switch` deosn't have `generic_new_flows`. Also, you need to map the flows into `GenericFlow` as highlighted here https://github.com/kytos-ng/flow_stats/pull/28#discussion_r948334431. @gretelliz, once your PR is ready, make sure to also to run it locally to confirm it's working as intended, if you mention that information that facilitates for reviewers too